tests/unit/knockout/editors/igCheckboxEditor/checkbox-test.js

```diff
@@ -0,0 +1,126 @@
+QUnit.module("Knockout unit tests for igCheckboxEditor", {
+	inputTag: '<input></input>',
+	util: $.ig.TestUtil,
+	editor: null,
+	model: null,
+	input: function() {
+		return this.editor.igCheckboxEditor("field");
+	},
+	container: function() {
+		return this.editor.igCheckboxEditor("editorContainer");
+	},
+	viewModel: function() {
+		this.isDisabled =  ko.observable(false);
+		this.isChecked = ko.observable(false);
+		this.isCheckedNonObservable = false;
+		this.isCheckedWithValue = ko.observable(false);
+		this.isCheckedNonObservableWithValue = false;
+	},
+	applyBindings: function() {
+		ko.applyBindings(this.model, this.qunitFixture[0]);
+	},
+	before: function () {
+		QUnit.config.reorder = false;
+		this.qunitFixture = $('#qunit-fixture');
+		this.model = new this.viewModel();
+	},
+	after: function () {
+		QUnit.config.reorder = true;
```

Comment:
Yeah..nope. The reorder config is false by default and is set on the adapter level, tests shouldn't change that.